src/pages/Projects.jsx

```diff
@@ -1,19 +1,172 @@
-import ProjectBox from '../components/projects/ProjectBox';
-import { Container } from '@mui/material';
+import projectsList from '../components/projects/projectsList';
+
+const renderingLinks = (href, icon) => {
+  return (
+    <Link
+      component={ReactRouterLink}
+      key={href}
+      to={href}
+      target="_blank"
+      rel="noopener noreferrer"
+      sx={linkStyle}
+    >
+      {icon}
+    </Link>
+  );
+};
+
+const boxStyle = (index) => ({
+  mb: { xs: 5, sm: 15 },
+  p: 5,
+  borderRadius: '25px',
+  minWidth: { xs: '95vw', sm: '80vw' },
+  maxWidth: { sm: '85vw' },
+  background:
+    index % 2 === 0
+      ? 'linear-gradient(270deg, rgba(217, 217, 217, 0) 38.54%, rgba(217, 217, 217, 0.4) 82.29%)'
+      : 'linear-gradient(90deg, rgba(217, 217, 217, 0) 38.54%, rgba(217, 217, 217, 0.4) 82.29%)'
+});
+
+const linkStyle = {
+  color: '#000',
+  textDecoration: 'none',
+  display: 'flex',
+  '&:hover': {
+    color: 'primary.main'
+  }
+};
+
+const projectGridLogo = (index, title, logo, links) => {
+  return (
+    <Grid item xs={12} md={6} order={{ xs: 0, md: index % 2 === 0 ? 2 : 1 }}>
+      {/* Logo */}
+      <CardMedia
+        component="img"
+        image={logo ?? null}
+        alt={`${title} logo`}
+        sx={{
+          mx: 'auto',
+          objectFit: 'contain',
+          maxWidth: '90%',
+          height: '150px'
+        }}
+      />
+      {/* Logo ends */}
+      {/* Links */}
+      <Stack direction="row" justifyContent="space-evenly" pt={{ xs: 0, sm: '1rem' }}>
+        {links.map(({ href, icon }) => renderingLinks(href, icon))}
+      </Stack>
+      {/* Links end */}
+    </Grid>
+  );
+};
+
+const projectGridContent = (index, description, title, status, techStack) => {
+  return (
+    <Grid item xs={12} md={6} order={{ xs: 0, md: index % 2 === 0 ? 1 : 2 }}>
+      <CardContent>
+        {/* Title and status */}
+        <Stack
+          direction={{ xs: 'column', sm: 'row' }}
+          spacing={1}
+          sx={{
+            alignItems: 'center',
+            justifyContent: 'space-between',
+            mb: '2rem'
+          }}
+        >
+          <Typography variant="h4" component="h3" sx={{ display: { xs: 'none', sm: 'block' } }}>
+            {title ?? null}
+          </Typography>
+          <Chip
+            label={status ?? null}
+            color="quaternary"
+            variant="outlined"
+            sx={{ backgroundColor: '#DEDEDEB2', fontWeight: 'bold' }}
+          />
+        </Stack>
+        {/* Title and status end */}
+        <Divider />
+        {/* Content */}
+        <Typography
+          variant="h6"
+          component="h4"
+          sx={{
+            fontWeight: 'bold',
+            mb: 2,
+            mt: 3,
+            textAlign: { xs: 'center', sm: 'left' }
+          }}
+        >
+          Overview
+        </Typography>
+        <Typography
+          variant="body1"
+          // TODO: Keep textAlign here?
+          sx={{ textAlign: 'justify' }}
```

Comment:
This weirdly gaps the text and we don't do that anywhere else. Don't see any reason to keep it.